### ISSUE TITLE
Error installing the apps runtime on i386

### ISSUE THREAD

**Issue by msalini** [tiliado/tiliado-repositories#24]
Hi there,

when I launch the flatpak install command, i get the following error:

```
flatpak install --from https://nuvola.tiliado.eu/eu.tiliado.Nuvola.flatpakref
This application depends on runtimes from:
  https://sdk.gnome.org/repo/
Configure this as new remote 'gnome' [y/n]: y
Installing: eu.tiliado.Nuvola/i386/stable
Warning: Can't find dependencies: No entry for app/eu.tiliado.Nuvola/i386/stable in remote summary flatpak cache 
Installing: eu.tiliado.Nuvola/i386/stable from eu.tiliado.Nuvola-origin
error: No such ref 'app/eu.tiliado.Nuvola/i386/stable' in remote eu.tiliado.Nuvola-origin
```

Is the i386 architecture supported?
Thanks
Marco

**Comment by jiri-janousek** [comment]
Hello @msalini. i386 is not supported, There should be a message about that on https://nuvola.tiliado.eu/. Is it shown on your PC?

**Comment by jiri-janousek** [comment]
![selection_003](https://user-images.githubusercontent.com/853706/32160591-af00c910-bd52-11e7-9f64-8be625250780.png)

**Comment by msalini** [comment]
Fenry,

That warning didn't show on my browser (maybe my noscript plugin was blocking it?) but thanks for pointing it out. 

Would you recommend waiting for the 32bit flatpak to become available (any ETA?) or roll back to 3.0?

Marco

**Comment by jiri-janousek** [comment]
> [...] (maybe my noscript plugin was blocking it?)

That might be the reason. The script tries to detect the browser architecture and shows the warning accordingly.

> Would you recommend waiting for the 32bit flatpak to become available (any ETA?) or roll back to 3.0?

I'd recommend upgrading to a 64bit system. Honestly, I doubt I'll get around to make 32bit flatpak builds unless someone funds that work. Nuvola 3 has less than 5% i386 downloads, so it just is not worth it.

**Comment by msalini** [comment]
My notebook has still plenty of life so I don't think I'd be willing to get rid of it only to be able to run Nuvola. But thanks for the suggestion ;-)
M.
